lib/devicetrust/friendly_enums.go

```diff
@@ -14,7 +14,10 @@
 
 package devicetrust
 
-import devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
+import (
+	devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
+	"github.com/gravitational/trace"
```

Comment:
```suggestion
	"github.com/gravitational/trace"

	devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
```

gci is bound to complain.